README.md

```diff
@@ -56,6 +56,7 @@ Here is the status of the various API components:
 * [Views](http://developer.zendesk.com/documentation/rest_api/views.html)
 * [Users](http://developer.zendesk.com/documentation/rest_api/users.html) ✓
 * [User Fields](https://developer.zendesk.com/rest_api/docs/core/user_fields.html) - Partial - List User Fields (`getUserField()`)
+* [User Related Information](https://developer.zendesk.com/rest_api/docs/core/users#user-related-information) ✓
```

Comment:
@caionovaes Maybe you could add it as a sub-item of Users

Reply:
@aheritier makes total sense but since I didn't see any other sub-items - ticket audit could be a sub-item of ticket - I stuck with the existing layout.